src/utils/panes/index.tsx

```diff
@@ -39,11 +39,11 @@ const useStyles = makeStyles({
     bottom: 16,
     position: 'absolute',
     right: 16,
-    top: 16,
+    top: '30vh',
     zIndex: 10,
   },
   paper: {
-    height: '100%',
+    height: 'calc(100vh - 32vh)',
```

Comment:
There's no need to use `calc()` for this because `calc(100vh - 32vh)` will always be the same value, and can be expressed as `68vh` instead. Use `calc()` when you have one relative value (e.g. `vh` or `%`) and one absolute value (e.g. `px`) and need to dynamically calculate the difference.

Reply:
Okay! I will leave it as it is for now!